src/main/asciidoc/reference/redis-repositories.adoc

```diff
@@ -711,6 +710,26 @@ The following table provides an overview of the keywords supported for Redis and
 |===============
 ====
 
+[[redis.repositories.queries.sort]]
+=== Sorting Query Method results
+
+Redis repositories allow various approaches to define sorting order. Redis itself does not support in-flight sorting when retrieving hashes or sets. Therefore, Redis repository query methods construct a `Comparator` that is applied to the result before returning results as `List`. Let's take a look at the following example:
+
+.Sorting Query Results
+====
+[source,java]
+----
+interface PersonRepository extends RedisRepository<Person, String> {
+
+  List<Person> findByFirstnameSortByAgeDesc(String firstname); <1>
```

Comment:
Shouldn't this be `OrderBy` instead of `SortBy`? The former is what the unit test uses.

Reply:
Good catch. This snippet originates from the MongoDB documentation that needs to be updated as well. Also, we should document the `OrderBy` keyword in the table of supported keywords.